.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,9 @@
+# Rails Zoo
+Congratulations! You're submitting your assignment!
+
+## Comprehension Questions
+| Question | Answer    |
+| :------------- | :------------- |
+| Describe in your own words the Rails request/response cycle for viewing an animals detail page |  |
+| Describe one area of Rails that you gained more clarity on with this assignment |    |
+| Is there a specific piece of code you'd like more feedback on? |  |
```

Comment:
Could we ask them some reflection questions like:
• What is one aspect of this project that surprised you? Why, and what did you do differently because of that?

or something along those lines?